src/pystatis/table.py

```diff
@@ -34,10 +36,20 @@ def get_data(self, area: str = "all", prettify: bool = True, **kwargs):
             area (str, optional): Area to search for the object in GENESIS-Online. Defaults to "all".
             prettify (bool, optional): Reformats the table into a readable format. Defaults to True.
         """
-        params = {"name": self.name, "area": area, "format": "ffcsv"}
+        params = {
+            "name": self.name,
+            "area": area,
+            "format": "ffcsv",
+            "language": "de",
```

Comment:
we should probably document that we set de as default language somewhere more prominent, like in the function's doc string but also in the general documentation